prisma/schema.prisma

```diff
@@ -91,8 +91,8 @@ model SocialRent {
 model GasBills {
   id     Int    @id
   region String @map("Region") @db.VarChar
-  itl    String @db.VarChar
-  bill   Float
+  itl1    String @db.VarChar
+  kwh_cost_pence   Float
```

Comment:
We also want to map this to camelCase 👍 

```suggestion
  kwhCostPence   Float @map("kwh_cost_pence")
```

Reply:
Still missing an `@map` here which would mean we can use camelCase not snake_case in `getGasBillByITL3()`

Reply:
Ah thanks for catching! Missed that one, should be fixed now at [07d2f98](https://github.com/theopensystemslab/fairhold-dashboard/pull/166/commits/07d2f98c6ebbc685c87612e25af2d2df1091e494)